pyproject.toml

```diff
@@ -144,8 +146,7 @@ testing-extra = [
     "ansi2html",
     "devtools",
     # used in docs tests
-    # TODO: remove bound once sqlalchemy supports 3.13
-    "sqlalchemy>=2.0,<3.0; python_version < '3.13'",
+    "sqlalchemy>=2.0,<3.0",
```

Comment:
I think we can only remove this bound if we change the lower bound for sqlalchemy?